package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
**[severity 1]: Package update**
            The updated package version might have breaking changes or new features, so it's important to test its compatibility with the project
            **How to fix:**
            Make sure to test the project compatibility with the updated package version